cli/src/cli.rs

```diff
@@ -3,8 +3,8 @@ use structopt::StructOpt;
 
 use crate::commands::{self, Command};
 use crate::errors::{ExitCode, Fallible};
-use crate::version::command_name;
 use crate::telemetry::Session;
+use crate::version::command_name;
```

Comment:
@Enrico2 I think this is another example to your point of us setting up linting

Reply:
++ @jsegaran and I have noticed there are distinct differences between what VSCode and IJ pick up for formatting. Would be nice to have a source of truth we can simply base our editors on.